avocado/core/safeloader.py

```diff
@@ -89,7 +89,10 @@ def is_docstring_directive_enable(docstring):
     :rtype: bool
     """
     result = get_docstring_directive(docstring)
-    return result == 'enable'
+    if 'enable' in result:
```

Comment:
The names begin to make less sense with this change.  I can certainly live with the current ones, but reading this code would be eaiser on **my mind** (YMMV) if this method was now called `docstring_directives_has_enable()`.

The implementation (suggestion) can also look like:

```python 
return 'enable' in get_docstring_directives(docstring)
```